app/src/main/java/com/android/shelfLife/model/recipe/RecipeRepositoryFirestore.kt

```diff
@@ -20,7 +21,6 @@ class RecipeRepositoryFirestore(private val db: FirebaseFirestore) : RecipeRepos
   }
 
   private val auth = FirebaseAuth.getInstance()
```

Comment:
this is not how you're supposed to get the user info. You need to ask for userID or whatever extra user information you need as a function parameter and when that function is called from a ViewModel you get the user data from the userRepositoryFirestore